SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,92 @@
+## Troubleshooting
+
+If an error occurs during the data fetching process, you can check the **Info** table to see a descriptive message about the error type. In this case, the **Sessions** table will contain the exact error record that you can inspect if you go to **Home > Transform data** and select the **Sessions** table.
+
+If you have already and successfully imported data from SPS and you run into an error when refreshing data so that the data is in the **Info** and **Sessions** table kept intact, you can find more details about the cause of the error in this guide.
+

```

Comment:
Here too, the loaded data will only be retained on refresh if an error is not returned on the first data query.

Reply:
See the [previous answer](https://github.com/OneIdentity/SafeguardPowerBI/pull/58#discussion_r1330579865).